std/random.d

```diff
@@ -415,8 +415,8 @@ $(D x0).
         return _x;
     }
 
-///
-    @property typeof(this) save() @safe pure nothrow
```

Comment:
It cannot be `BigInt`. `isUnsigned` is only true for built-in types which are unsigned integer types, and that's what's used on `UintType` in the template constraints. So, please put back the attributes that you removed. And feel free to add `@nogc` if it works. None of those attributes are going to depend on whether the template argument is a `ulong`, `uint`, etc. If `BigInt` and other user-defined types were allowed, then yes, attribute inference would be needed, but they're not, so it's not.